### DISCUSSION TITLE
How do I hide an element below 640px and show it on wider screens?

### DISCUSSION THREAD

**Discussion by mosemet** [tailwindlabs/tailwindcss#2079] [Help] [answered]
Hi

How can I hide a component for screens below 640px. sm:hidden only works for screens 640px and above.

**Comment by gsimone** [comment, marked as answer]
`<div class="hidden sm:block" />` will make the element visible up to the sm breakpoint. Tailwind is mobile-first

**Reply by mosemet** [reply]
Thank you
